Src/Support/GoogleApis.Auth.PlatformServices_Shared/OAuth2/GoogleCredential.cs

```diff
@@ -148,38 +148,25 @@ public GoogleCredential CreateScoped(params string[] scopes)
             return CreateScoped((IEnumerable<string>) scopes);
         }
 
-        #region IConfigurableHttpClientInitializer
```

Comment:
Because regions are *really annoying*, especially when they are this small and pointless.